client/src/extension.ts

```diff
@@ -188,6 +190,27 @@ export function deactivate(): Thenable<void> | undefined {
   }
   return Promise.all(promises).then(() => undefined);
 }
+
+const didApplyRefactoringCommandId = "elm.didApplyRefactoring";
```

Comment:
I know in other places you had to add an `extensionId` postfix. Is this necessary here? I don't feel like it should be and I'm not sure how to do it either.

Reply:
You need to pass the `workspaceId` to the function registering. I'll have a look at the branch.

Reply:
Ah, right, the other reference is the problem.

Reply:
Maybe we just need to add a check to make sure this command only gets registered once?

Reply:
We can try, but I don't think that's enough

Reply:
Can you explain why?

Reply:
I think I'm wrong, from testing, doing 

```
if (!isRegistered) {
          registerDidApplyRefactoringCommand(context);
          isRegistered = true;
        }
```

seems to work fine. The code before also seemed to work fine, but had an error in the debug console.

Reply:
Or instead can we just pull it outside of the forEach loop?

Reply:
Makes way more sense :)